ppr-ui/src/composables/useInputRules.ts

```diff
@@ -55,13 +55,23 @@ export const useInputRules = () => {
   }
 
   const isEmail = (): Array<Function> => [
-    (v: string) => !!v || 'Email address is required',
+    (v: string) => !!v || 'Enter an email address',
```

Comment:
Could you confirm that we are updating this error message for all email fields across the app?

Reply:
I will check with UX.

I did a search across ppr for occurances of isEmail and it does not seem to be called anywhere else.